src/BSSelectIntegrationService/RetrieveMeshFile/Program.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using NHS.MESH.Client;
+using Common;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWebApplication()
```

Comment:
Don't think we need the ASP.NET Core integration so let's replace this with `ConfigureFunctionsWorkerDefaults()` to align with our other functions